### PR TITLE
[artifactory] [artifactory-ha] Artifactory ha add custom labels to pods

### PR DESCRIPTION
**Pull request by franpb90**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:  Add primary and node labels also to pods and services.

**Special notes for your reviewer**: By default labels from values are only set in controllers, and we need these labels in pods and services because we are going to use them for monitoring.

### CODE REVIEW

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [3.0.6] - Jul 15, 2020
+* Updated Artifactory-ha Chart to add node and primary labels to pods and service objects.
```
Comment (chukka) [inline stable/artifactory-ha/CHANGELOG.md:5]: @franpb90  can you add a line space

**Comment by chukka** [comment]
@franpb90  Thanks for contribution ! Can you please add the same changes for `artifactory` chart in the same PR

File: stable/artifactory-ha/templates/artifactory-node-statefulset.yaml [hunk 2]
```diff
@@ -30,6 +30,9 @@ spec:
         heritage: {{ .Release.Service }}
         component: {{ .Values.artifactory.name }}
         release: {{ .Release.Name }}
+        {{- if .Values.artifactory.node.labels }}
```
Comment (danielezer) [inline stable/artifactory-ha/templates/artifactory-node-statefulset.yaml:33]: Use a with clause please

File: stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml [hunk 3]
```diff
@@ -34,6 +34,9 @@ spec:
         component: {{ .Values.artifactory.name }}
         heritage: {{ .Release.Service }}
         release: {{ .Release.Name }}
+        {{- if .Values.artifactory.primary.labels }}
```
Comment (danielezer) [inline stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml:37]: Use a with clause please

File: stable/artifactory-ha/templates/artifactory-service.yaml [hunk 4]
```diff
@@ -9,6 +9,9 @@ metadata:
     component: {{ .Values.artifactory.name }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    {{- if .Values.artifactory.node.labels }}
```
Comment (danielezer) [inline stable/artifactory-ha/templates/artifactory-service.yaml:12]: Use a with clause please

File: stable/artifactory-ha/templates/artifactory-service.yaml [hunk 5]
```diff
@@ -65,6 +68,9 @@ metadata:
     component: {{ .Values.artifactory.name }}
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
+    {{- if .Values.artifactory.primary.labels }}
```
Comment (danielezer) [inline stable/artifactory-ha/templates/artifactory-service.yaml:71]: Use a with clause please

**Comment by franpb90** [comment]
Hi, 

Once the pull-request have passed all checks, What I can do to merge it to master? When it passed all the checks, it says that I do not have permission to complete it because I do not have contributor permission.

Thank you,

**Comment by danielezer** [comment]
We will merge it when it will pass all the tests :)

**Pull request merged**